### PR TITLE
Add tire skidding sound

### PR DESCRIPTION
**Pull request by nima3333**
Linked to https://github.com/heclak/community-a4e-c/issues/471

Thanks to @callmepartario who provided the assets and helped me a lot, this is a first try to implement the skidding tires of the A4.

It's working by getting, for each gear, the wheel angular position. An array of 10 (arbitrary) positions is stored. Then, if the elements are all identical, the gear has WoW and the horizontal speed is more than 1 m/s (arbitrary), a skidding detector param is set.

The param is directly used by the sound system in a new mode I've written for the occasion, SOUND_ONCE_STOPABLE. The problem with SOUND_ONCE was that the sound effect would not stop if the breaks were released in the middle of it. With the new mode, the sound will stop and can be replayed.

For the sdef and the assets themselves, @callmepartario did a great job.

Added: do not trigger the skidding on carriers

### CODE REVIEW

File: A-4E-C/Cockpit/Scripts/Systems/gear.lua [hunk 1]
```diff
@@ -89,7 +89,92 @@ dev:listen_command(device_commands.emer_gear_release)
...
+for i=1, 10 do
+    no_anim_L_array[i] = -1
+    no_anim_R_array[i] = -1
+    no_anim_N_array[i] = -1
+end
+local skid_L_param = get_param_handle("SKID_L_DETECTOR")
+local skid_R_param = get_param_handle("SKID_R_DETECTOR")
+local skid_N_param = get_param_handle("SKID_N_DETECTOR")
+
+function get_base_sensor_data()
```
Comment (08jne01) [inline A-4E-C/Cockpit/Scripts/Systems/gear.lua:106]: Why is this here? Maybe you are looking for a similar function which can be found in EFM_data_bus

Comment (nima3333) [reply A-4E-C/Cockpit/Scripts/Systems/gear.lua:106]: Oh, it is here because I copy-paste the ```on_carrier``` function from ```Cockpit\Scripts\Systems\carrier.lua``` (with some modifications), and this function was defined inside it (```carrier.lua```). After review, only using the default ```get_base_data``` from EFM_data_bus should be fine, let me modify it.

Comment (nima3333) [reply A-4E-C/Cockpit/Scripts/Systems/gear.lua:106]: Done, let me know if it is now ok for you (tested in quick missions on land and aircraft carriers)

Comment (08jne01) [reply A-4E-C/Cockpit/Scripts/Systems/gear.lua:106]: You can just do something like: 

`local sensor_data = get_efm_sensor_data_overrides()`

at the top. sensor_data will just be a table of functions so you only need to call it once to get the table.

Then wherever you want the position
`local px,py,pz = sensor_data.getSelfCoordinates()`

This would simplifiy this.

Comment (08jne01) [reply A-4E-C/Cockpit/Scripts/Systems/gear.lua:106]: You actually already have and use a sensor_data at the top see line 27

Comment (nima3333) [reply A-4E-C/Cockpit/Scripts/Systems/gear.lua]: Got it, have made the change to use the already declared sensor_data
